app/tests/database.py

```diff
@@ -0,0 +1,140 @@
+from stac_api.models import Asset
+from stac_api.models import Collection
+from stac_api.models import CollectionLink
+from stac_api.models import Item
+from stac_api.models import ItemLink
+from stac_api.models import Keyword
+from stac_api.models import Provider
+from stac_api.models import get_default_stac_extensions
+
+
+def create_collection():
+    '''Create a dummy collection db object for testing
+    '''
+    collection = Collection.objects.create(
+        crs=["http://www.opengis.net/def/crs/OGC/1.3/CRS84"],
+        created='2020-10-28T13:05:10.473602Z',
+        updated='2020-10-28T13:05:10.473602Z',
+        description='This is a description',
+        start_date=None,
+        end_date=None,
+        extent=[200000, 100000, 200001, 100005],
+        collection_name='ch.swisstopo.pixelkarte-farbe-pk200.noscale',
+        item_type='Feature',
+        license='test',
+        stac_extension=get_default_stac_extensions(),
+        stac_version="0.9.0",
+        summaries_eo_gsd=[10.1, 20.3, 30.44],
+        summaries_proj=[1, 4, 22],
+        geoadmin_variant=['blubb', 'blabb', 'blibb'],
```

Comment:
Not compatible with the latest collection model. Summaries are now stored in JSONField.